FSharp.Profiles.props

```diff
@@ -15,7 +15,6 @@
     <DefineConstants>$(DefineConstants);FX_NO_ARRAY_LONG_LENGTH</DefineConstants>
```

Comment:
There are several defines in here that *are* in NET Standard 2.0. For example, LongLength, threading, binary serialization, and probably way more.

Reply:
@saul, I Plan on doing a full pass before I merge this.  However, there is a pretty weird assembly resolution issue due to this change, that I need to work through first.

Kevin